src/main/java/ch/naviqore/raptor/router/RouteScanner.java

```diff
@@ -388,7 +389,128 @@ private boolean checkIfTripIsPossibleAndUpdateMarks(int targetTime, ActiveTrip a
         return null;
     }
 
+    /**
+     * Get the trip offsets for a stop in a given range.
+     * <p>
+     * This method gets all departure / arrival offsets for a stop in a given range required to scan to catch all trips
+     * departing / arriving at stop in range (time type is set in route scanner object). The reference time for the stop
+     * is the time of the first label of the stop (in round 0). This method gets all stop times for each route departing
+     * / arriving at the stop in the range and calculates all the trip offsets from the reference time. The trip offsets
+     * are expressed in a positive number of seconds from the reference time (in case of arrival scanning these values
+     * should be subtracted from the latest arrival time).
+     *
+     * @param stopIdx the index of the stop to get the trip offsets for.
+     * @param range   the range to get the trip offsets for.
+     * @return the trip offsets for the stop in the given range.
```

Comment:
See Docstring further above:

> The trip offsets are expressed in a positive number of seconds from the reference time (in case of arrival scanning these values should be subtracted from the latest arrival time).